templates/class_template.erb

```diff
@@ -0,0 +1,11 @@
+# class_template.erb
+# frozen_string_literal: true
+
+class Scarpe
```

Comment:
If we do templates, we probably also want display properties, plus binding an event.

Reply:
do we need binding an event for all? like for fill,background we dont ig?

Reply:
should i do like in comment?

Reply:
We definitely don't need bindings -- or maybe even display properties -- for literally every widget/object. In a comment would be fine for those, sure. It's a template, so it's good to include optional-but-common stuff too.

Reply:
Yeah I love the idea of having the options commented out. If you are familiar with Rails you can see examples of this in config/production.rb - though that's just one example of a nice and common idiom :)